facts/get_facts.sh

```diff
@@ -311,6 +316,8 @@ bundle install --path vendor/bundler
 for version in 1.6.0 1.7.0 2.0.0 2.1.0 2.2.0 2.3.0 2.4.0 2.5.0; do
   FACTER_GEM_VERSION="~> ${version}" bundle update
   case "${operatingsystem}" in
+    rocky)
+	;&
```

Comment:
why is this different from almalinux below?

Reply:
`;&` is fallthrough, so it's exactly the same as almalinux

Reply:
It is more important that code is consistent and `;;` is what is already used and is normal for bash.

Reply:
While `;&` seems to be portable, I have to admit I never saw it in use before and had to check the man page to see if it was portable.

In my experience, the above code is generally "seen in the wild" as follow, and is I guess less surprising:

```sh
case "${operatingsystem}" in
  almalinux|rocky)
    ...
    ;;
esac
```

Reply:
> I [...] had to check the man page to see if it was portable.

My man pages are lying to me… `;&` is not part of the [POSIX Shell Command Language](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18).